new-frontiers-14/frontier-station-14
# GameRule Rework (POI Spawning)

## About the PR


Complete overhaul of the back end of POI spawning.
## Why / Balance

It started with 1 single grid and grew to like 20 why did we ever let it get this bad
## How to test

It should just work like normal. Load up in release and start a new round, check all the POIs spawned like usual.
## Media

![geode 2](https://github.com/user-attachments/assets/d7bca154-2c3c-4a55-8a2b-3ed7ef79625b)
![geode 1](https://github.com/user-attachments/assets/66bc8ec4-ad9e-44dd-bcea-5823117b06b5)
![geode 3](https://github.com/user-attachments/assets/04d72466-7bf4-4ebb-8f64-2c9ba11a4a97)
![depotalt](https://github.com/user-attachments/assets/add6c014-c81e-44b7-91dc-e28f6177a42f)


## Breaking changes


**Changelog**



:cl:
- tweak: Reworked POI Spawning
